services/visual_recognition/v3.js

```diff
@@ -155,6 +163,12 @@ module.exports = function (RED) {
       apikey = sAPIKey || this.credentials.apikey;
       this.status({}); 
 
+      visualRecognition = watson.visual_recognition({
+        api_key: apikey,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'api_key' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2714287336/issues/source?bid=3330694&fileBranchId=3344618#l167)